src/neuroconv/basedatainterface.py

```diff
@@ -20,12 +24,16 @@ def get_source_schema(cls):
         """Infer the JSON schema for the source_data from the method signature (annotation typing)."""
         return get_schema_from_method_signature(cls, exclude=["source_data"])
 
-    def __init__(self, **source_data):
+    def __init__(self, verbose: bool = True, **source_data):
         self.source_data = source_data
+        self.verbose = verbose
 
     def get_conversion_options_schema(self):
         """Infer the JSON schema for the conversion options from the method signature (annotation typing)."""
-        return get_schema_from_method_signature(self.run_conversion, exclude=["nwbfile", "metadata"])
+        try:
+            return get_schema_from_method_signature(self._run_conversion, exclude=["nwbfile", "metadata"])
+        except ValueError:
```

Comment:
I'd change to excepting only the `NotImplementedError`; if there's a separate error in the schema inference (but private method is defined), wouldn't want to fall back to the wrong signature